jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/admin/AdminRunSteps.java

```diff
@@ -50,4 +50,10 @@ public void verifyTextAreaItem(final String itemName, final String textAreaHeadi
         final String val = webElement.getAttribute("value");
         assert StringUtils.isNotEmpty(val) && val.contains(itemName);
     }
+
+    @And("^I see client details \"(.*)\" \"(.*)\"")
```

Comment:
Just make the statement more precise e.g. 

    I see in the web banner that the client name = \"(.*)\", HHRR = \"(.*)\" and data mode = \"(.*)\"

With regards to the HHRR let's put a sample HHRR for our 49 environment USMLVV1CDS0049.json

https://github.cerner.com/malvern-hs/chef-stp/pull/101/files#diff-973b0972f6f9f251aeafaa5e6c1ebd42R70